amundsen_application/static/js/components/common/ResourceList/ResourceListHeader/index.tsx

```diff
@@ -5,21 +5,40 @@ import * as React from 'react';
 
 import './styles.scss';
 
+import { ResourceType } from 'interfaces';
 import {
   RESOURCE_HEADER_TITLE,
   SOURCE_HEADER_TITLE,
   BADGES_HEADER_TITLE,
+  LAST_UPDATED_HEADER_TITLE,
 } from './constants';
 
-const ResourceListHeader: React.FC = () => {
+export interface ResourceListHeaderProps {
+  resourceTypes: ResourceType[];
+}
+
+const contentHeaderTitle = (type: ResourceType): string => {
+  switch (type) {
+    case ResourceType.dashboard:
+      return LAST_UPDATED_HEADER_TITLE;
+
+    default:
+      return BADGES_HEADER_TITLE;
+  }
+};
+const ResourceListHeader: React.FC<ResourceListHeaderProps> = ({
+  resourceTypes,
+}: ResourceListHeaderProps) => {
+  const contentHeader =
+    resourceTypes.length === 1 ? contentHeaderTitle(resourceTypes[0]) : '';
   return (
     <div className="resource-list-header">
       <span className="resource">
         <span className="resource-text">{RESOURCE_HEADER_TITLE}</span>
       </span>
       <span className="source">{SOURCE_HEADER_TITLE}</span>
       <span className="badges">
-        <span className="badges-text">{BADGES_HEADER_TITLE}</span>
+        <span className="badges-text">{contentHeader}</span>
```

Comment:
It could be that we don't have a badge text but we still render this element, right?

Reply:
Correct; I was worried that the layout might get messed up if we exclude the entire tree, but I can change to do that?

Reply:
Good call, I don't know what would happen. If you could test it, that would we awesome, thanks!

Reply:
ah, it makes the middle column float on over to the right.
<img width="856" alt="Screen Shot 2020-12-02 at 10 33 13 AM" src="https://user-images.githubusercontent.com/217874/100915982-faee0980-3489-11eb-8c49-c926ac9a0625.png">

Reply:
Yeah, let's leave it empty for now!